### PR TITLE
ACN_CD_JuniperSRX_DataConnector01

### PR DESCRIPTION
**Pull request by chicduong**
## Proposed Changes

  - Juniper SRX Data Connector
  - Juniper SRX Sample Data

### CODE REVIEW

**Review by v-jayakal — DISMISSED** [review]

File: DataConnectors/Juniper SRX/Connector_Syslog_JuniperSRX.json [hunk 1]
```diff
@@ -0,0 +1,123 @@
+{ 
+    "id": "JuniperSRX",
+    "title": "Juniper SRX", 
+    "publisher": "Juniper", 
+    "descriptionMarkdown": "The [Juniper SRX](https://www.juniper.net/us/en/products-services/security/srx-series/) connector allows you to easily connect your Juniper SRX logs with Azure Sentinel This gives you more insight into your organization's network and improves your security operation capabilities.",
```
Comment (nazang) [inline DataConnectors/Juniper SRX/Connector_Syslog_JuniperSRX.json:5]: typo

'.' is missing

File: DataConnectors/Juniper SRX/Connector_Syslog_JuniperSRX.json [hunk 2]
```diff
@@ -0,0 +1,123 @@
+{ 
+    "id": "JuniperSRX",
+    "title": "Juniper SRX", 
+    "publisher": "Juniper", 
+    "descriptionMarkdown": "The [Juniper SRX](https://www.juniper.net/us/en/products-services/security/srx-series/) connector allows you to easily connect your Juniper SRX logs with Azure Sentinel This gives you more insight into your organization's network and improves your security operation capabilities.", 
+    "additionalRequirementBanner":"These queries and workbooks are dependent on a parser based on a Kusto Function to work as expected. Follow the steps to use this Kusto functions alias **JuniperSRX** in queries and workbooks. [Follow these steps to get this Kusto functions.](https://aka.ms/sentinelgithubparsersjunipersrx)",
+    "graphQueries": [
```
Comment (nazang) [inline DataConnectors/Juniper SRX/Connector_Syslog_JuniperSRX.json:7]: @chicduong, could you update the link to  https://aka.ms/sentinel-junipersrx-parser

For future, all the shortlink should be

https://aka.ms/sentinel-[connectorid]-parser  for parser, and
https://aka.ms/sentinel-[connectorid]-azuredeploy for .zip raw azure function.

@v-jayakal, we need to provide the correct link based on the naming schema. Please update the documentation for these as well.

**Review by nazang — CHANGES_REQUESTED** [review]

File: DataConnectors/Juniper SRX/Connector_Syslog_JuniperSRX.json (hunk 1 above)
Comment (chicduong) [reply DataConnectors/Juniper SRX/Connector_Syslog_JuniperSRX.json]: updated

File: DataConnectors/Juniper SRX/Connector_Syslog_JuniperSRX.json (hunk 2 above)
Comment (chicduong) [reply DataConnectors/Juniper SRX/Connector_Syslog_JuniperSRX.json:7]: updated and noted for future connectors

**Review by nazang — APPROVED** [review]

**Pull request merged**